test/classes/dinan/init_before_userdef_constructor.chpl

```diff
@@ -1,19 +0,0 @@
-proc f(x) {
```

Comment:
And maybe this one?  I might be good to do a greptests for the path name to the tests in these commits, since I may miss a couple